my-code/movie/src/index.css

```diff
@@ -0,0 +1,244 @@
+:root {
+  /* Colors */
+  --yellow: #ff9f1c;
+  --red: #ff4040;
+  --green: #2ec4b6;
+  --dark: #0a1014;
+  --grey: #1b2329;
+  --mid-grey: #353f4c;
+  --light-grey: #7a8c99;
+  --white: #fff;
+}
+
+@import url('https://fonts.googleapis.com/css?family=Roboto:400,500,700');
+
+html, body {
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+	height: 100%;
+	font: calc(100% + 0.1vw)/1.5 Roboto, -apple-system, system-ui, BlinkMacSystemFont, "Segoe UI";
+  /* font: 16px/1.5 Roboto, -apple-system, system-ui, BlinkMacSystemFont, "Segoe UI"; */
+  font-weight: 400;
+  background: var(--dark);
+  color: var(--white);
+}
+
+a, 
+a:link, 
+a:visited,
+a:hover,
+a:active {
+  text-decoration: none;
+  color: inherit;
+}
+
+.m-0 {
+  margin: 0;
+}
+
+.btn {
+  display: inline-block;
+  text-align: center;
+  white-space: nowrap;
+  vertical-align: middle;
+  box-sizing: border-box;
+  border: 1px solid var(--mid-grey);
+  border-radius: 5px;
+  font-size: 1rem;
+  padding: 0.5em 0.75em;
```

Comment:
Don't mess up the units, choose only one type.
I like a lot the idea about `em` and `rem`, and I used to set for `font-size` the `rem` (root/body `em`) and paddings and margins as `em`.

Reply:
That was probabilly a typo and was suposed to be rem